benchmark/tangurnis/matlab.m

```diff
@@ -28,7 +34,7 @@
 
         k = vv(1,3)*pi; %1*pi
         a=vv(1,4); % 0 or 2
-        fprintf('computing Di=%i, gamma = %i\n', Di, gamm)
+        fprintf('computing Di=%i, gamma = %i, a= %i\n', Di, gamm, a)
```

Comment:
Who wrote this originally that they couldn't spare the extra letter to call it `gammA`? ;-)